Calculator2/src/Calculator2.java

```diff
@@ -0,0 +1,61 @@
+import java.util.Scanner;
+
+public class Calculator2 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("Enter the first number");
+        if (sc.hasNextDouble()) {
+            double firstValue = sc.nextDouble();
+            System.out.println("Input is correct");
+
+
+            System.out.println("Enter an operation (for example: +, -, *, /, %)");
+            String operation = sc.next();
+            switch (operation) {
+                case "+", "-", "*", "/", "%" -> System.out.println("User inputs supported operations");
```

Comment:
ahah, looks interesting) Please avoid using switch case for such cases

Reply:
Done. I have changed 'switch' to 'if'